dspy/teleprompt/copro_optimizer.py

```diff
@@ -331,7 +329,7 @@ def compile(self, student, *, trainset, eval_kwargs):
                 )
 
             if self.prompt_model:
-                logger.debug(f"{self.prompt_model.inspect_history(n=1)}")
+                self.prompt_model.inspect_history(n=1)
```

Comment:
We can probably just remove this logging, I suspect if users will be able to interpret the overwhelming logging...